Makefile.calicoctl

```diff
@@ -7,6 +7,12 @@
 # Determine which OS / ARCH.
 OS := $(shell uname -s | tr A-Z a-z)
 ARCH := amd64
+###############################################################################
+# Subcomponent versions:
+GO_BUILD_VER:=latest
+# Set libcalico-go version in glide.yaml first then do a `glide up -v` to update glide.lock.
+LIBCALICOGO_VER := v1.0.0
```

Comment:
Is LIBCALICOGO_VER suppose to match up with the version glide.yaml is updated to? If so, the comment should probably state that explicitly.  I also wanted to check that LIBCALICOGO_VER is not suppose to be used anywhere in the Makefiles besides just to be printed out.